contracts/LosslessStaking.sol

```diff
@@ -187,6 +187,7 @@ contract LosslessStaking is ILssStaking, Initializable, ContextUpgradeable, Paus
 
         (,,,, ILERC20 reportTokens,,) = losslessReporting.getReportInfo(_reportId);
 
+        // TODO: Update staking amount on LssGov
```

Comment:
No need to update cause the it's just that user will receive less tokens, but if we transfer out 50, we transfer out 50 and that's it, nothing changes on our side?

Reply:
Correct, no need to update. At the end the problem was very simple, we just needed to start taking into consideration the amount after the `retrieveBlacklistedFunds` After that, everything runs smoothly because the fee is not being taken as an extra. Is being deducted from the amount the receiver receives.

Removed lingering TODOs on a410114